client.go

```diff
@@ -12,11 +12,17 @@ import (
 	"strconv"
 	"time"
 
+	"runtime/debug"
+
 	"github.com/mdempsky/gocode/gbimporter"
 	"github.com/mdempsky/gocode/suggest"
 )
 
 func doClient() {
+	// Client is a short-lived program.
+	// Disable GC to make it faster
+	debug.SetGCPercent(-1)
```

Comment:
I am not fan of this myself, but it does speed up autocompletion about 30% (from 38-40ms to 26-28ms on a simple test). This actually makes client side autocompletion faster than server side.